in_text/text_06_habitat.yml

```diff
@@ -119,7 +131,7 @@ entities:
         attr-def: Year of the summarized temperature predictions
         attr-defs: This data release
         data-min: 1980
-        data-max: 2018
+        data-max: 2019
```

Comment:
huh, do these have 2019 data in them? of yeah...realizing this has a different range than MNTOHA 🤦

Reply:
Yep! PB0 ends on `2019-08-21` and PBALL ends on `2019-08-07`